### DISCUSSION TITLE
How to get list of all dependencies needed for a package to work?

### DISCUSSION THREAD

**Discussion by pwhitener** [composer/composer#11042] [Q&A / Support]
Is there a way to get a list of all dependencies needed for certain package to work?

Let's say I need some package and this package has dependencies. Then each of these dependencies also have dependencies.

Is there a way to get the list of all dependencies that are needed for a package to work? Not just the ones in `composer.json`of that package

**Comment by Seldaek** [comment]
I guess the easiest way would be to create a new composer.json with only that package, then `composer update --no-install && composer show --locked` should give you a list of those packages.
